M2/Macaulay2/m2/exports.m2

```diff
@@ -60,6 +60,7 @@ export {
 	"AssociativeExpression",
 	"Authors",
 	"AuxiliaryFiles",
+	"BF",
```

Comment:
I'd rather keep this, as well as `IT`, because it makes substituting `\bf{...}` to `BF{...}` very straightforward:
https://github.com/mahrud/M2/blob/b6b561ba1155b9201b558873839737641f7e2d21/M2/Macaulay2/m2/hypertext.m2#L228-L237

I tried to do this another way without exporting BF and IT, perhaps with locally defining them, but it didn't work. If you have a suggestion to make that work then we can unexport them.